model_templates/python3_sklearn_runtime_params/custom.py

```diff
@@ -4,47 +4,55 @@
 This is proprietary source code of DataRobot, Inc. and its affiliates.
 Released under the terms of DataRobot Tool and Utility Agreement.
 """
+import logging
+
 # Use this helper class to access the runtime parameter values in your model
 from datarobot_drum import RuntimeParameters
 
 
+logger = logging.getLogger(__name__)
+
+
 # This is a naive function so as to not dump the full credential values
 # during this demonstration.
 def mask(value, visible=3):
     return value[:visible] + ("*" * len(value[visible:]))
 
 
 def transform(data, model):
-    print("=" * 40)
-    print("Loading the following Runtime Parameters:")
+    logger.info("=" * 40)
     option1 = RuntimeParameters.get("option1")
-    print(f"\toption1: {option1}")
     option2 = RuntimeParameters.get("option2")
-    print(f"\toption2: {option2}")
     option3 = RuntimeParameters.get("option3")
-    print(f"\toption3: {option3}")
+    logger.info(
+        "Loading the following Runtime Parameters: "
+        f"option1: {option1}, option2: {option2}, option3: {option3}",
+    )
 
     credential = RuntimeParameters.get("encryption_key")
     if credential is not None:
         credential_type = credential.pop("credentialType")
-        print(
-            f"\tapi_key(type={credential_type}): "
-            + str({k: mask(v) for k, v in credential.items()})
+        logger.info(
+            "Using credentials api_key: ",
+            extra={
+                "credential_type": credential_type,
+                "api_key": str({k: mask(v) for k, v in credential.items()}),
+            },
         )
     else:
-        print("No credential data set")
+        logger.info("No credential data set")
 
     # boolean runtime param
     bool_var = RuntimeParameters.get("bool_var")
-    print(f"\tbool_var: {bool_var}")
+    logger.info("\tbool_var: %s", bool_var)
```

Comment:
I guess, these `\t` chars are not needed in case of logging? Or we still want to format strings with them?

Reply:
This log will be also shown up when using cli. logging supports tab whitespaces so it should be good to preserve original formatting

Reply:
Sounds fair 👍